File: Java/Program/Product.java
/*Saya Muhammad Rifky Afandi dengan NIM 2202346 mengerjakan LP2 dalam mata kuliah 
Desain Pemrograman Berbasis Objek untuk keberkahanNya maka saya tidak 
melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin.*/

public class Product {
    //private attribute
    private String idProduct;
    private String name;
    private String brand;
    private String price;

    //Constructor without parameter
     public Product(){
        this.idProduct = "";
        this.name = "";
        this.brand = "";
        this.price = "";
    }

    //constructor with parameter
    public Product(String idProduct, String name, String brand, String price){
        this.idProduct = idProduct;
        this.name = name;
        this.brand = brand;
        this.price = price;
    }

    /*Getter dan Setter */
    public void setiDProduct(String idProduct){
            this.idProduct = idProduct;
    }

    public String getIdProduct(){
        return this.idProduct;
    }

    public void setName(String name){
        this.name = name;
    }

    public String getName(){
        return this.name;
    }

    public void setBrand(String brand){
        this.brand = brand;
    }

    public String getBrand(){
        return this.brand;
    }

    public void setprice(String price){
        this.price = price;
    }

    public String getprice(){
        return this.price;
    }


    
}

File: Python/Program/Product.py
## Saya Muhammad Rifky Afandi dengan NIM 2202346 mengerjakan LP2 dalam mata kuliah 
## Desain Pemorograman Berbasis Objek untuk keberkahanNya maka saya tidak 
## melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin.

## Membuat class
class Product:
    ## Atribut privat
    __id = ""
    __name = ""
    __brand = ""
    __price = ""
    
    ## Constructor
    ## Atribut Publik
    def __init__(self, id = "", name = "", brand = "", price = ""):
        self.__id = id
        self.__name = name
        self.__brand = brand
        self.__price = price
        
    ## Getter dan Setter
        
    def get_id(self):
        return self.__id
    
    def set_id(self, id):
        self.__id = id
        
    def get_name(self):
        return self.__name
    
    def set_name(self, name):
        self.__name = name
        
    def get_brand(self):
        return self.__brand
    
    def set_brand(self, brand):
        self.__brand = brand
    
    
    def get_price(self):
        return self.__price
    
    def set_price(self, price):
        self.__price = price
    
    